### ISSUE TITLE
TypeError with Django 1.11

### ISSUE THREAD

**Issue by ataylor32** [dominicrodger/django-tinycontent#20]
I created a test project to demonstrate this issue. All I did was add `tinycontent` to my `INSTALLED_APPS` and replace the contents of my urls.py with this:

```python
from django.conf.urls import url
from django.http import HttpResponse
from django.template import Context, Template


def test_view(request):
    template = Template(
        '{% load tinycontent_tags %}'
        '{% tinycontent_simple "Test" %}'
    )
    return HttpResponse(template.render(Context()))

urlpatterns = [
    url(r'^$', test_view),
]
```

This is what happens:

```
Internal Server Error: /
Traceback (most recent call last):
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/tinycontent/templatetags/tinycontent_tags.py", line 63, in tinycontent_simple
    obj = TinyContent.get_content_by_name(content_name)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/tinycontent/models.py", line 33, in get_content_by_name
    obj = TinyContent.objects.get(name=name)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/db/models/query.py", line 379, in get
    self.model._meta.object_name
tinycontent.models.DoesNotExist: TinyContent matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/tinycontenttest/project/project/urls.py", line 11, in test_view
    return HttpResponse(template.render(Context()))
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/library.py", line 203, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/tinycontent/templatetags/tinycontent_tags.py", line 78, in tinycontent_simple
    context)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/backends/django.py", line 64, in render
    context = make_context(context, request, autoescape=self.backend.engine.autoescape)
  File "/var/www/tinycontenttest/venv/lib/python3.4/site-packages/django/template/context.py", line 287, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than Context.
```

The same code works with Django 1.10.7.

**Comment by dominicrodger** [comment]
Thanks for the report @ataylor32 - this is fixed in 6b40e3f59f4b25b09a956335ba9a50b2b1a77848. I'll get a release pushed shortly.

**Comment by dominicrodger** [comment]
Uploaded to PyPI as version 0.7.0. Let me know if you see any issues - thanks again!
